catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -337,6 +344,12 @@
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Validation.VALIDATORS,
+                true /* indexed */,
```

Comment:
If we're going to make this queryable, we should make sure it works. Can you add an itest that ingests a metacard that fails a validator with errors and warnings, warnings, and neither and then query for them back out based on if the validator had errors, warnings, and neither? I'd like to see that it works and that the `<boolean, boolean>` tuple works.